client/verta/requirements.txt

```diff
@@ -1,7 +1,7 @@
 click >= 7.0, < 9.0
 cloudpickle >= 1.0, < 3.0
 googleapis-common-protos >= 1.5, < 2.0
-importlib_metadata >= 3.7.0
+importlib-metadata >= 3.7.0
```

Comment:
Switched to a hyphen to be consistent with how the package is listed in PyPI: https://pypi.org/project/importlib-metadata/

Doesn't seem to make a difference for `pip`, though.